src/jspdf.js

```diff
@@ -3132,9 +3196,21 @@ function jsPDF(options) {
       throw new Error("Invalid unit: " + unit);
   }
 
+  var encryption = null;
   setCreationDate();
   setFileId();
 
+  var getEncryptor = (API.__private__.getEncryptor = API.getEncryptor = function(
```

Comment:
I think we should expose `getEncryptor` only to the `internal` API.

Reply:
alright